force-app/main/default/objects/Journey__c/fields/Candidate_Email_Address__c.field-meta.xml

```diff
@@ -1,13 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
     <fullName>Candidate_Email_Address__c</fullName>
-    <caseSensitive>true</caseSensitive>
     <description>Stores Mainlydigital&apos;s candidate email address.</description>
     <externalId>false</externalId>
     <label>Candidate Email Address</label>
     <required>true</required>
     <trackHistory>false</trackHistory>
     <trackTrending>false</trackTrending>
     <type>Email</type>
-    <unique>true</unique>
```

Comment:
This field could have been left as unique because its unlikely a candidate will have multiple simultaneous journeys. I only complained on the `Manager Email` field because one manager is likely to manage multiple candidates.

I'm not asking you to change it back again but I just wanted to make sure my reasoning makes sense to you.